Formula/haskell-language-server.rb

```diff
@@ -62,7 +66,7 @@ def caveats
     <<~EOS
       #{name} is built for GHC versions #{ghc_versions}.
       You need to provide your own GHC or install one with
-        brew install ghc
+        brew install #{newest_ghc}
```

Comment:
```suggestion
        brew install #{ghcs.last}
```
Could fix in a follow-up, though.